src/schema/SchemaDefinition.ts

```diff
@@ -64,12 +64,7 @@ export interface RawInlinePropDefinition {
 export interface ParsedInlinePropDefinition {
 	name: string;
 	raw?: undefined;
-	content: InlinePropContentDefinition[];
-}
-
-export interface InlinePropContentDefinition {
-	schema: string;
-	tag: string;
+	content: ReadonlySet<string>;
```

Comment:
I'm somewhat against this change, for the following reasons:

- Schemas can no longer be loaded to and from JSON
- It's harder to write schemas. For instance, a tag that takes no content could previously be written as `content: []`. With this change, it becomes `content: new Set<string>()`
- If we want to treat the content as a set internally, we can do that in the `Schema` class rather than impose this at the to the user, at the edge of the system.

Reply:
Done ✅